core-services/release-controller/_releases/priv/release-ocp-4.8.json

```diff
@@ -44,6 +44,13 @@
                 "name": "release-openshift-ocp-installer-e2e-aws-serial-4.8-priv"
             }
         },
+        "aws-single-node": {
```

Comment:
disabled jobs don’t need to be added here to show up in testgrid.

if your job is periodic, you can just skip this.

Reply:
It is periodic but the periodicity is yearly, the goal was to have it triggered when new release images are created by adding it to the release controller. Is that not the way to do it?

Reply:
Change your periodic to not be yearly.  In general, we are stopping the habit of running every test on every payload because we already have to look at way more jobs than can run on the payload.  I will probably be pruning down the nightly runs to representative samples (in which case we'll want one) but you want to have good periodics (24 or 48h), not expect to run on every payload just yet.

Reply:
Got it. So I'll close this PR for now (will re-open in the future if needed) and create a [new PR](https://github.com/openshift/release/pull/16601) with daily periodic